magpie/api/notifications.py

```diff
@@ -23,6 +23,10 @@
 
 TEMPLATE_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), "templates")
 DEFAULT_TEMPLATE_MAPPING = {
+    "MAGPIE_GROUP_TERMS_APPROVED_EMAIL_TEMPLATE":
+        os.path.join(TEMPLATE_DIR, "email_group_terms_approved.mako"),
+    "MAGPIE_GROUP_TERMS_SUBMISSION_EMAIL_TEMPLATE":
+        os.path.join(TEMPLATE_DIR, "email_group_terms_submission.mako"),
```

Comment:
These need to be added to docs since they can be overridden by configuration. 
Probably a new section in https://raw.githubusercontent.com/Ouranosinc/Magpie/master/docs/configuration.rst under the `User Registration and Approval Configuration` section.
Must also be added to below `get_email_template` function.

Reply:
I'll add template variables in the magpie.ini also, like you did for the user registration templates.